dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletAPIInterceptor.java

```diff
@@ -883,6 +883,22 @@ public Contentlet findContentletByIdentifierAnyLanguage(String identifier) throw
 		return c;
 	}
 
+	@Override
+	public Contentlet findContentletByIdentifierAnyLanguage(String identifier, String variant) throws DotDataException {
```

Comment:
it seems to be a new method, not sure if any language is a good one, since received a variant